openmsupply/open-msupply
# 1954 UI prescription list view

Fixes #1954 

# 👩🏻‍💻 What does this PR do? 
List view for Prescriptions (buttons and row click currently lead to blank page. This will be fixed in another issue).
New `Prescription` link under `Dispensary` and shortcut to go to prescriptions.

# 🧪 How has/should this change been tested? 
- Create some dummy prescriptions in OG or with GraphQL endpoint in omSupply
- Make sure OG `EditPatient` permission is on
- Store will need to be a dispensary
- Go to Dispensary > Prescriptions to see view of all prescription invoices
